test/assets/PropertyDetailsTestConstants.scala

```diff
@@ -60,7 +61,8 @@ object PropertyDetailsTestConstants {
     firstAccountingPeriodEndDate = None,
     incomeSourceType = None,
     tradingStartDate = None,
-    latencyDetails = None
+    latencyDetails = None,
```

Comment:
can we add unit tests with examples of conversion for the fields added, something like:
`  val testMinimumPropertyDetailsJson = Json.obj(
    "incomeSourceId" -> "111111111111111",
    "accountingPeriodStartDate" -> "2017-06-01",
    "accountingPeriodEndDate" -> "2018-05-31",
    "latencyDetails" -> ... ,
     "cashOrAccrualsFlag" -> true
  )`